### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by SimonHeybrock**
Tutorials taken from recent scipp course at IKON, with only minor changes.

### CODE REVIEW

File: docs/data/fetch_neutron_data.py [hunk 1]
```diff
@@ -25,13 +25,20 @@ def download_file(source, destination):
...
+        "3d543bc6a646e622b3f4542bc3435e7e",
+        "LARMOR00049338.nxs":
+        "https://github.com/ess-dmsc-dram/loki_tube_scripts/raw/master/test/test_data/LARMOR00049338.nxs",  # noqa: E501
     }
 
     for f, h in data_files.items():
         destination = os.path.join(target_dir, f)
         if not os.path.isfile(destination):
-            download_file(os.path.join(remote_url, h), destination)
+            url = h if h[:4] == 'http' else os.path.join(remote_url, h)
```
Comment (nvaytet) [inline docs/data/fetch_neutron_data.py:43]: I usually prefer `startswith('http')` to specifying the substring length, but it's really not that important.

File: docs/tutorials/exploring-data.ipynb [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Use the HTML representation to see what the loaded data contains\n",
+    "\n",
+    "The HTML representation is what Jupyter displays for a scipp object.\n",
+    "- Take some time to explore this view and try to understand all the information (dimensions, dtypes, units, ...).\n",
+    "- Note that sections can be expanded, and values can shown by clicking the icons to the right."
```
Comment (nvaytet) [inline docs/tutorials/exploring-data.ipynb:60]: I think i suggested this before: we could add the unicode symbol &#9921; there to make it more clear which symbols we are talking about.
But you didn't include it last time, so maybe you have a good reason?

File: docs/tutorials/exploring-data.ipynb [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+   "outputs": [],
+   "source": [
+    "data.attrs['proton_charge_by_period'].value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exercises:\n",
```
Comment (nvaytet) [inline docs/tutorials/exploring-data.ipynb:137]: "Exercises" in bold?

File: docs/tutorials/exploring-data.ipynb [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+    "1. Find some attributes of `data` with `dtype=DataArray` and plot their `value`.\n",
+    "   Also try `sc.table(attr.value)` to show a table representation.\n",
+    "2. Find and plot a monitor.\n",
+    "3. Try to normalize `data` to monitor 1.\n",
+    "   Why does this fail?\n",
+    "4. Plot all the monitors on the same plot.\n",
+    "   Note that `sc.plot()` can be used with a Python `dict` for this purpose: `sc.plot({'a':something, 'b':else})`.\n",
+    "5. Convert all the monitors from `'tof'` to `'wavelength'` using, e.g., `mon1_wav = sc.neutron.convert(mon1, 'tof', 'wavelength')`.\n",
+    "6. Inspect the HTML view and note how the \"unit conversion\" changed the dimensions and units.\n",
+    "7. Re-plot all the monitors on the same plot, now in `'wavelength'`."
```
Comment (nvaytet) [inline docs/tutorials/exploring-data.ipynb:147]: Should we also suggest to normalize again to monitor after the conversion to wavelength has been done?

**Review by nvaytet — COMMENTED** [review]
I see this has been merged already, but since you asked for comments i reviewed anyway ;-)

File: docs/tutorials/exploring-data.ipynb (hunk 2 above)
Comment (SimonHeybrock) [reply docs/tutorials/exploring-data.ipynb:60]: I did include it last time, but it showed a different symbol, so it was removed again.

**Pull request merged**